packages/augur-core/source/contracts/trading/ZeroXTrade.sol

```diff
@@ -69,13 +72,15 @@ contract ZeroXTrade is Initializable, IZeroXTrade, IERC1155 {
     // solhint-disable-next-line var-name-mixedcase
     bytes32 public EIP712_DOMAIN_HASH;
 
+    IAugur augur;
     IFillOrder public fillOrder;
     ICash public cash;
     IShareToken public shareToken;
     IExchange public exchange;
 
     function initialize(IAugur _augur, IAugurTrading _augurTrading) public beforeInitialized {
         endInitialization();
+        augur = _augur;
```

Comment:
You don't appear to actually use this, did you mean to add it?

Reply:
Its used to check allowances in the balanceOf calls

Reply:
oh thats a different PR actually... Well it _will_ be doing thast heh

Reply:
Nitpick: Consider moving this change to that other PR then.  😄  If not, see comment above about modifier.